packages/ecs-browser/tsup.config.js

```diff
@@ -4,8 +4,8 @@ export default defineConfig({
   entry: ["src/index.ts"],
   outDir: "dist",
   format: ["esm"],
+  dts: false,
   sourcemap: true,
   clean: true,
-  dts: true,
-  noExternal: ["goober", "shiki"],
```

Comment:
not sure if it's necessary anymore to force these into the bundle, so I'm trying without them for now

but we'll likely remove this package from v2 anyway